File: source code/UOSense-Backend/src/main/java/UOSense/UOSense_Backend/repository/ReportRepository.java
package UOSense.UOSense_Backend.repository;

import UOSense.UOSense_Backend.entity.Report;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ReportRepository extends JpaRepository<Report, Integer> {
    void deleteAllByReviewId(int reviewId);
}

File: source code/UOSense-Frontend/app/src/main/java/com/example/uosense/data/ReportRepository.kt
package com.example.uosense.data

import com.example.uosense.models.Report

object ReportRepository {

    fun getReports(): List<Report> {
        // API 또는 DB 연동 시 실제 데이터로 대체
        return listOf(
            Report("홍길동", null, "★★★★★", "리뷰 제목 1", "리뷰 내용 1", "2024-12-06"),
            Report("김철수", null, "★★★★☆", "리뷰 제목 2", "리뷰 내용 2", "2024-12-05"),
            Report("이영희", null, "★★★☆☆", "리뷰 제목 3", "리뷰 내용 3", "2024-12-04")
        )
    }
}
